src/inausoft.netCLI.Tests/LogicalCommandDeserializerTests.cs

```diff
@@ -91,6 +135,22 @@ public void RegexArgumentHandler_ThrowsDeserializationException_ForInvalidOption
             Assert.AreEqual(ErrorCode.UnrecognizedOption, exception.ErrorCode);
         }
 
+        [TestMethod]
+        public void RegexArgumentHandler_ThrowsDeserializationException_WhenOptionIsSuppliedWithOneHyphen()
```

Comment:
Rename the first part to proper method name (`Deserialize_hrowsDeserializationException_WhenOptionIsSuppliedWithOneHyphen`)

Reply:
Adjusted.